R/evaluate-clusters.R

```diff
@@ -206,10 +208,14 @@ calculate_stability <- function(
   # ensure we have a matrix
   pca_matrix <- prepare_pc_matrix(x, pc_name = pc_name)
 
-  # check clusters and matrix compatibility
+  # Extract vector of clusters, ensuring same order as pca_matrix
+  rownames(cluster_df) <- cluster_df$cell_id
+  clusters <- cluster_df[rownames(pca_matrix),]$cluster
```

Comment:
> My only other thought is that you might want to make the data frame more flexible, with the default column being cluster but allow it to be anything? That change would have to propagate through the other functions too, presumably, and I would expect it to be a separate issue and PR.

I extremely had this thought too while working on this package with very fresh eyes. It would be good to set `cell_id` and `cluster` as default in arguments for this column names. I'll open an issue.

Reply:
I think leaving `cell_id` as required is fine, as that is part of the SCE spec, and we can require whatever we want. But I guess up to you.